peewee.py

```diff
@@ -4413,6 +4412,14 @@ def bind(self, model, name, set_attribute=True):
     def column(self):
         return Column(self.model._meta.table, self.column_name)
 
+    @property
+    def has_default_constraint(self):
+        if self.constraints:
+            for constraint in self.constraints:
+                if constraint.sql.startswith(("DEFAULT", "default")):
```

Comment:
this is a little too specific. e.g. what if the constraint used `"Default"`. I would just compare against `constraint.sql.lower()`